ci-operator/jobs/infra-periodics.yaml

```diff
@@ -1241,6 +1241,10 @@ periodics:
       - --ignore-repo=ci-operator/jobs/openshift/cluster-api-provider-kubevirt
       - --ignore-org=openshift-priv
       - --ignore-org=kubernetes-sigs
+      - --ignore-org=ci-operator/config/layered-products
```

Comment:
you need layered-products or cspi only? I do not see "layered-products" in the PR, it's for the future?

Reply:
leftover, removed